sequence_processing_pipeline/Pipeline.py

```diff
@@ -293,72 +292,33 @@ def _configure_profile(self):
 
                 # the 'profile' attribute must have a dictionary as its value.
                 # all profiles must contain 'instrument_type' and 'assay_type',
-                # unless instrument_type == 'default', in which case the
-                # profile defines the defaults across all instrument-types and
-                # assay-types.
                 if 'instrument_type' not in contents['profile']:
                     raise ValueError("'instrument_type' is not an attribute "
                                      f"in '{profile_path}'.profile")
 
                 if 'assay_type' not in contents['profile']:
-                    if contents['profile']['instrument_type'] != 'default':
-                        raise ValueError("'assay_type' is not an attribute "
-                                         f"in '{profile_path}'.profile")
+                    raise ValueError("'assay_type' is not an attribute "
+                                     f"in '{profile_path}'.profile")
 
                 profiles.append(contents)
 
-        # The default profile provides 'fall-through' configuration settings
-        # for all items. This allows the user to not have to redefine settings
-        # for all items for all instrument and assay combinations.
-
-        # the final profile is created by taking the default profile and using
-        # it as a base. If a profile matching the run-directory's instrument
-        # and assay types is found, settings from that profile will overwrite
-        # the base-profile settings as appropriate.
-        base_profile = None
         selected_profile = None
 
-        # iterate through all the profiles, searching for a default
-        # profile and the first profile w/matching instrument and assay types.
-        # if a matching profile isn't found, that's okay, but if a default
-        # profile isn't found, then raise an Error.
-
         for profile in profiles:
-            p_i_type = profile['profile']['instrument_type']
-            if p_i_type == 'default':
-                base_profile = profile
-            else:
-                p_a_type = profile['profile']['assay_type']
+            i_type = profile['profile']['instrument_type']
+            a_type = profile['profile']['assay_type']
 
-                # if both items have been found, it's safe to break early.
-                if base_profile is not None and selected_profile is not None:
-                    break
+            print("PROFILE FOUND: %s, %s" % (i_type, a_type))
+
+            if i_type == instr_type and a_type == assay_type:
+                selected_profile = profile
+                break
+
+        if selected_profile is None:
+            raise ValueError(f"a matching profile ({instr_type}, {assay_type}"
```

Comment:
maybe add to report it to admin? (AKA us)

Reply:
We haven't introduced email notifications into this part of the code, so IMOO we should handle it as a separate issue. I think we would be better served with a chunk of code that catches all (or just many) Errors and sends an email along with displaying the stack trace to the user like we already do.

Reply:
ha! I meant add to the message that they should email the admins and not just ignore